main.go

```diff
@@ -80,6 +88,11 @@ func (c *rootCmd) Run(fl *pflag.FlagSet) {
 		dir = "~"
 	}
 
+	// Get linux relative path if on windows.
+	if runtime.GOOS == "windows" {
+		dir = gitbashWindowsDir(dir)
+	}
+
```

Comment:
This should be handled in sshcode.go, not main.go.

Reply:
Which function should i put it in inside sshcode.go? as there are multiple and main.go is the primary function so it is used for everything.

Reply:
It should be near the top of sshCode(), probably above the ssh master connection code.

Reply:
bump

Reply:
i swear i fixed this already WTF?

Reply:
Okay so upon looking at it, i may be wrong but i dont think its a good idea to move this.
because if i move it out from here

`dir` is declared right before this, And then `dir` is used right after, we dont want one part of this to get the wrong `dir`var. 
Will look deeper into it however

```go
	dir := fl.Arg(1)
	if dir == "" {
		dir = "~"
	}

	// Get linux relative path if on windows.
	if runtime.GOOS == "windows" {
		dir = gitbashWindowsDir(dir)
	}

	err := sshCode(host, dir, options{
		skipSync:         c.skipSync,
		sshFlags:         c.sshFlags,
		bindAddr:         c.bindAddr,
		syncBack:         c.syncBack,
		reuseConnection:  !c.noReuseConnection,
		uploadCodeServer: c.uploadCodeServer,
	})
```

Reply:
yeah `sshCode()` is complicated enough as it is.

Reply:
yeah im not going to change this as it is helpful to keep things simpler so there is less chance of a future commit involving an issue where it screws with something